packages/hub/src/scope.ts

```diff
@@ -185,12 +186,20 @@ export class Scope implements ScopeInterface {
   /**
    * @inheritDoc
    */
-  public setTransaction(transaction?: string): this {
-    this._transaction = transaction;
+  public setTransactionName(name?: string): this {
```

Comment:
offtopic: This whole transaction thing will be very confused (and possibly broken) if someone uses `Sentry.Integrations.Transaction` in node env.